targets/ar71xx-generic/profiles.mk

```diff
@@ -289,9 +289,9 @@ $(eval $(call GluonModel,OMEGA,onion-omega,onion-omega))
 
 # Hornet-UB
 $(eval $(call GluonProfile,HORNETUB))
-$(eval $(call GluonModel,HORNETUB,hornet-ub,alfa-hornet-ub))
-$(eval $(call GluonModelAlias,HORNETUB,alfa-hornet-ub,alfa-ap121))
-$(eval $(call GluonModelAlias,HORNETUB,alfa-hornet-ub,alfa-ap121u))
+$(eval $(call GluonModel,HORNETUB,alfa-hornet-ub,alfa-network-hornet-ub))
```

Comment:
Yes, only the last argument may be changed. "alfa-network-hornet-ub" is correct.